cdap-docs/developers-manual/source/data-exploration/streams.rst

```diff
@@ -135,17 +135,23 @@ For example::
 CSV and TSV Formats
 -------------------
 The ``csv`` (comma separated values) and ``tsv`` (tab separated values) formats read event bodies as delimited text.
-They have two settings, ``charset`` for the text charset, and ``delimiter`` for the delimiter.
+They have three settings, ``charset`` for the text charset, ``delimiter`` for the delimiter, and ``mapping`` for
+column index to schema field mapping.
```

Comment:
They have three settings: ``charset`` for the text charset; ``delimiter`` for the delimiter; and ``mapping`` for
column-index-to-schema-field mapping.